jsonnetlib/k8s.jsonnet

```diff
@@ -152,6 +152,7 @@ cfg {
     env=[],
     container={},
     podSpec=root.podSpec,
+    ttl=180,
```

Comment:
Could you move this into `config.jsonnet`?

Reply:
Also, I feel 3 minutes is a bit short as a default value, is there any recommendation? 30 minutes?

Reply:
I don't see recommendations and I guess 30m is fine